### ISSUE TITLE
Letzte Version nicht kompatibel mit Contao 4 (siehe screenshot)

### ISSUE THREAD

**Issue by mem-lima** [ErdmannFreunde/contao-grid-bundle#23]
Ich kann mit Contao 4.13.26 und PHP 8.1 das mit Contao-Manager nicht installieren. Ich müsste manuell die Version 1.1.3 einstellen. Hat dann geklappt..

![image](https://github.com/erdmannfreunde/contao-grid-bundle/assets/118195454/d920d092-1aca-4e0f-a793-2b5e2b5fdec9)

**Comment by denniserdmann** [comment]
Ja, das ist richtig so. Die Version 2 ist ausschließlich für Contao 5.

**Comment by mem-lima** [comment]
Alles klar, ich denke nur, dass wenn man Contao 4.13 installiert hat, sollte man trotzdem die Erweiterung installieren können, ohne dass man die Version manuell anpassen soll? mhh..

**Comment by denniserdmann** [comment]
Ich kläre das, inwieweit wir dem Manager diese Information geben können. Es wird noch weitere Extensions betreffen, deswegen bin ich zuversichtlich, dass es auf kurz oder lang eine Lösung geben wird.

**Comment by katgirl** [comment]
Nach meinem Verständnis müsste es reichen in der '''composer.json''' die 4er-Version als Konflikt anzugeben für die Version 2.0. Dann sollte der Manager und auch composer eigentlich die 1.x installieren.

**Comment by denniserdmann** [comment]
Ich habe den Konflikt in der Version 2.0.5 ergänzt. Ich habe gerade keine frische Contao Installation zum Testen, würde mich also über Feedback freuen.

**Comment by denniserdmann** [comment]
Problem ist behoben und nicht mehr relevant, da Contao 4.13 EOL ist.
